src/components/PageWrapper.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactNode } from 'react'
+
+// eslint-disable-next-line react/display-name
+export default ({ children }: { children: ReactNode }): JSX.Element => (
```

Comment:
can we use `export default const PageWrapper = () => {}`

as eslint suggested?